web/src/engine/websites/ReadComicOnline.ts

```diff
@@ -13,19 +14,36 @@ function ChapterExtractor(anchor: HTMLAnchorElement) {
     };
 }
 
-function MangInfoExtractor(element: HTMLDivElement) { //needed because titles are truncated in link
-    const anchor = element.querySelector<HTMLAnchorElement>('a');
-    const dom = new DOMParser().parseFromString(element.getAttribute('title'), 'text/html');
-    return {
-        id: anchor.pathname,
-        title: dom.querySelector('p.title').textContent.trim()
-    };
-}
+const pageScript = `
+    new Promise( (resolve, reject) => {
+        const arrayName = document.documentElement.innerHTML.match(/(\\w+)\\.push\\(pth/)[1];
+        let tries = 0;
+        const interval = setInterval(function () {
+            try {
+                const imagearray = window[arrayName];
+                if (!Array.isArray(imagearray)) return;
+                const lastElement = imagearray[imagearray.length - 1];
+                const url = !/blank.gif/i.test(lastElement) && new URL(lastElement);
+                if (url) {
+                    clearInterval(interval);
+                    resolve (imagearray);
+                }
+            } catch (error) {
+                tries++;
```

Comment:
This catch block will never be reached in case the last image is always a valid URL containing `blank.gif`

Reply:
Should i put a reduced timeout for the promise then ?

Reply:
It is ore a re-arranging issue. Solution maybe to move the code into a finally block, or just place it after the catch block.